docs/docs/60-new-docs/40-operator-guide/20-advanced-installation/20-advanced-with-argocd.md

```diff
@@ -4,6 +4,160 @@ sidebar_label: With Argo CD
 
 # Advanced Installation with Argo CD
 
+This section outlines the various advanced installation options available for deploying Kargo on a Kubernetes cluster using Argo CD.
+
+:::warning
+This section assumes that you have already installed any dependencies or prerequisites required for running Kargo on a Kubernetes cluster. Please see the [installation guide](../../40-operator-guide/10-basic-installation.md#prerequisites) for more information.
+:::
```

Comment:
Only for consistency, I might make this a note instead of a warning. The way I've been using the "severity" levels here is:

* info == here's more information if you're interested (generally safe to skip)
* note == you should probably look at this
* warning/caution == there's danger ahead if you _don't_ look at this

I think the danger here is minimal since it's just going to fail the installation and not irreparably break anything, although I'm willing to be convinced otherwise.

```suggestion
:::note
This section assumes that you have already installed any dependencies or prerequisites required for running Kargo on a Kubernetes cluster. Please see the [installation guide](../../40-operator-guide/10-basic-installation.md#prerequisites) for more information.
:::
```

Reply:
I was actually wondering how we define these note "levels". I lean on just keeping it consistent with the rest of the documentation; and since I don't have a strong opinion I'm just going to follow convention here.